src/Microsoft.PowerShell.Commands.Utility/commands/utility/TestJsonCommand.cs

```diff
@@ -137,10 +194,36 @@ protected override void BeginProcessing()
         protected override void ProcessRecord()
         {
             bool result = true;
+            string resolvedPath = string.Empty;
+            string jsonToParse = string.Empty;
+
+            if (Json != null)
+            {
+                jsonToParse = Json;
+            }
+
+            if (Path != null)
+            {
+                resolvedPath = PathUtils.ResolveFilePath(Path, this, _isLiteralPath);
+
+                if (!System.IO.File.Exists(resolvedPath))
```

Comment:
`System.IO` is redundant.
```suggestion
                if (!File.Exists(resolvedPath))
```

Reply:
Fixed